rayon-core/src/scope/mod.rs

```diff
@@ -346,3 +347,14 @@ impl<'scope> Scope<'scope> {
         }
     }
 }
+
+impl<'scope> fmt::Debug for Scope<'scope> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        use latch::LatchProbe;
+        fmt.debug_struct("Scope")
+            .field("owner_thread", &self.owner_thread)
+            .field("panic", &self.panic)
+            .field("job_completed", &self.job_completed_latch.probe())
```

Comment:
I think `probe()` will always be true until the scope is out of the user's hands.  For debugging purposes, the actual counter value would be more interesting.  How about adding `Debug` to `CountLatch` and show that here?

Reply:
alright